stat/stat.go

```diff
@@ -20,6 +20,8 @@ type CumulantKind int
 const (
 	// Empirical treats the distribution as the actual empirical distribution.
 	Empirical CumulantKind = 1
+	// LinInterp uses a linear interpolation on the empirical distribution.
```

Comment:
Maybe

```
LinInterp linearly interpolates the empirical distribution between sample values, where small p are given the first value of x, and p=1 is the last value of x
```

I'm not picky about the exact wording, I mostly want the documentation to convey that this interpolation is direction asymmetric.

Reply:
Maybe a clearer way to express `where small p are given the first value of x, and p=1 is the last value of x`  would be `with a flat extrapolation`?